docs/en/news/2025-11-05-crowdfunding-feature-pinned-posts-in-groups/README.md

```diff
@@ -0,0 +1,91 @@
+---
+home: false
+article: true
+sidebar: false
+lang: en-US
+date: 2025-11-05
+category:
+  - Funding
+  - Features
+tag:
+  - Funding
+  - Groups
+  - Posts
+  - Pins
+cover: /blog/crowdfunding-feature-pinned-posts-in-groups--en--corrected.png
+coverAlt: "Our first crowdfunding campaign!"
+title: "Our first crowdfunding campaign! 🪄✨"
+description: "Help with your donation, toenable pinning posts in groups."
+---
+
+<!-- markdownlint-disable no-inline-html first-line-heading -->
+
+Help with your donation, toenable pinning posts in groups.
+
+<DonationBar
+  :currentValue="690"
+  :target="1200"
+  startDate="2025-11-05"
+  endDate="2026-01-02"
+  asOfDate="2025-11-17"
+/>
+
+### What it’s about
+
+We are now launching our first crowdfunding campaign for all users of *ocelot.social* networks and for anyone interested in promoting independent social networks.
+
+We are starting with a feature that ensures that important information remains more visible in groups.
+
+### Pinning posts in groups
+
+The goal is to allow group administrators to pin posts to the top of their group’s post list,
+so that members can see these announcements immediately.
+This feature already exists for the general post list.
+To make it available for groups, this feature needs to be significantly expanded and programmatic tests need to be added for quality assurance.
+
+<figure class="blog-post-image">
+  <img src="/blog/pinned-post-in-group--en.png" alt="Pinned post on group page" />
+  <figcaption class="blog-post-image-capture">
+    This is what a pinned post on a group page could soon look like (orange frame and banner).
+  </figcaption>
+</figure>
+
+### Why a crowdfunding?
+
+busFaktor() e.V., the organization behind the free open-source software *ocelot.social*, is a non-profit.
+Since it has no commercial interests, the further development of the software is financed solely through volunteer work, donations, membership fees, and contracts from the operators of *ocelot.social* networks to freelance developers.
+This includes donations from you.
+
+<!-- markdownlint-disable no-duplicate-heading -->
+<DonationBar
+  :currentValue="690"
+  :target="1200"
+  startDate="2025-11-05"
+  endDate="2026-01-02"
+  asOfDate="2025-11-17"
+/>
+
+### Donations
+
+We welcome your donation to:
+
+busFaktor() e.V.
+IBAN: DE81 5003 1000 1084 5340 01
+BIC: TRODDEF1
+Triodos Bank N.V., Germany
+
+Location: 86179 Augsburg, Germany
+
+For the purpose of use, see below.
+
+### Notes on the purpose of use
+
+Please be sure to include "Donation" and your email address in the purpose of use so that we can contact you. The "@" symbol is not normally possible in bank transfers, so please write it as "at", with a space before and after it and without quotation marks.
```

Comment:
```suggestion
Please be sure to include "Donation" and your e-mail address in the purpose of use so that we can contact you. The "@" symbol is not normally possible in bank transfers, so please write it as " at ", with a space before and after it and without quotation marks.
```